smart_open/http.py

```diff
@@ -30,7 +30,7 @@ class BufferedInputBase(io.BufferedIOBase):
     """
 
     def __init__(self, url, mode='r', buffer_size=DEFAULT_BUFFER_SIZE,
-                 kerberos=False, user=None, password=None):
+                 kerberos=False, user=None, password=None, headers=None):
```

Comment:
The default headers are simple, so why not do?

```suggestion
                 kerberos=False, user=None, password=None, headers=_HEADERS):
```

Reply:
-1 on that (for the same reason mentioned above -- mutable parameter defaults lead to subtle bugs).